package.json

```diff
@@ -8,11 +8,13 @@
     "nuke": "asini clean && rm -r node_modules",
     "changelog": "asini-changelog",
     "debug": "cat asini-debug.log && for d in packages/*/npm-debug.log*; do echo $d; cat $d; done",
-    "postinstall": "npm run bootstrap-no-i"
+    "postinstall": "npm run bootstrap-no-i",
+    "update-dependencies": "asini exec david u"
```

Comment:
Should this also run david on the repo root?  Then we could disable the GreenKeeper integration and just manage everything with david?